src/screens/EditorScreen/components/ContentEditor/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { useState, useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import ReactQuill from 'react-quill';
+import 'react-quill/dist/quill.snow.css';
+import cn from 'classnames';
+
+import { FORMATS, MODULES } from './constants';
+import styles from './styles.module.scss';
+
+interface Props {
+  isPostbodyLongEnough: boolean;
+  postBody: string;
+  setPostBody: React.Dispatch<React.SetStateAction<string>>;
+}
+
+function ContentEditor({ isPostbodyLongEnough, postBody, setPostBody }: Props) {
+  const { t } = useTranslation('EditorScreen');
+  const [isFocused, setIsFocused] = useState(false);
+  const [hasBeenBlurred, setHasBeenBlurred] = useState(false);
+  const [postBodyLength, setPostBodyLength] = useState(1);
+  const doesPostBodyHaveLength = postBodyLength > 1;
+  const isError = hasBeenBlurred && !isPostbodyLongEnough;
+
+  const handleChangeSelection = (
+    selection: ReactQuill.Range,
+    _source: string,
+    editor: ReactQuill.UnprivilegedEditor
+  ) => {
+    if (selection) {
+      setIsFocused(true);
+    } else {
+      setIsFocused(false);
+    }
```

Comment:
I can't do it that way because the event fires after each key stroke as well, triggering a focus state change after every key stroke.

![pr_newpost_fail1](https://user-images.githubusercontent.com/62221379/131386830-482d87c0-0f2d-40ad-9dae-6aea1be53962.gif)